src/main/java/uk/gov/pay/api/app/config/RedisConfiguration.java

```diff
@@ -20,14 +20,22 @@
     private boolean ssl;
     
     @Valid
-    @JsonProperty("timeout")
-    private Duration timeout;
+    @JsonProperty("commandTimeout")
+    private Duration commandTimeout;
+
+    @Valid
+    @JsonProperty("connectTimeout")
+    private Duration connectTimeout;
 
     public String getUrl() {
         return format("%s://%s", ssl ? "rediss" : "redis", endpoint);
     }
 
-    public Long getTimeout() {
-        return timeout.toMilliseconds();
+    public Long getCommandTimeout() {
```

Comment:
Is there a reason to convert these to milliseconds as `Long`'s rather than leave them as `Duration` types? Where they're used they are converted back into `Duration` types?

Reply:
`RedisConfiguration` holds Dropwizard `Duration` types which are then converted via `Long` to Java `Duration` types